curriculum/js2/3.js

```diff
@@ -5,9 +5,28 @@
  * @param {array} a
  * @returns {array}
  */
+ const isPrime=(num,i=2)=>{
```

Comment:
Is it possible for you to import it instead?

`const isPrime = require('../js3/4.js').solution // just add the correct file path to the isPrime solution`